src/devices/third_reality.ts

```diff
@@ -237,14 +237,13 @@ const definitions: Definition[] = [
         vendor: 'Third Reality',
         description: 'Zigbee multi-function night light',
         ota: ota.zigbeeOTA,
-        fromZigbee: extend.light_onoff_brightness_colortemp_color().fromZigbee.concat([
-            fzLocal.thirdreality_private_motion_sensor, fz.illuminance, fz.ias_occupancy_alarm_1_report]),
-        toZigbee: extend.light_onoff_brightness_colortemp_color().toZigbee,
-        exposes: [e.light_brightness_colortemp_color([153, 555]).removeFeature('color_temp_startup'),
+        fromZigbee: [fz.color_colortemp, fz.on_off, fz.brightness, fz.level_config, fz.ignore_basic_report,
```

Comment:
You can use `extend.light_onoff_brightness_color` instead

Reply:
OK, thanks. I will retry.